lib/segment/src/index/field_index/full_text_index/inverted_index.rs

```diff
@@ -20,47 +20,87 @@ impl Document {
 }
 
 pub struct ParsedQuery {
-    pub tokens: BTreeSet<String>,
+    pub tokens: Vec<Option<usize>>,
 }
 
 impl ParsedQuery {
     pub fn check_match(&self, document: &Document) -> bool {
+        if self.tokens.contains(&None) {
```

Comment:
What is this `None` check for?

Reply:
Yes, that's right. Sorry for the late response , didn't catch the email earlier.